js/aframe/index.js

```diff
@@ -8,15 +8,38 @@
  */
 
 var viewer = {};
+viewer.y = 0;
 
 viewer.createViewUnknown = function(model)
 {
   let view;
   var value = viewer.el.getAttribute('text', value).value;
   if (model.type) {
-    value += "\nTODO: " +  model.type;
+    value += `\n${model.name} (${model.type})`;
   }
-  viewer.el.setAttribute('text', 'value', value);
+  //viewer.el.setAttribute('text', 'value', value);
+
+  // https://github.com/gmarty/aframe-ui-components
+  // https://github.com/caseyyee/aframe-ui-widgets
+  // https://github.com/rdub80/aframe-gui
+  var newEl = document.createElement( "a-sphere" )
+  newEl.setAttribute( "position", "0 " + ( viewer.y += 0.3 ) + " 0" )
+  newEl.setAttribute( "radius", "0.1" )
+  newEl.setAttribute( "color", "lightblue" )
+  var readwrite = true
+  if ( readwrite ){
```

Comment:
maybe we need to define a style and use same everywhere

Reply:
Your choice, I have no preference